SPECS/glibc/glibc.spec

```diff
@@ -257,6 +264,7 @@ grep "^FAIL: nptl/tst-eintr1" tests.sum >/dev/null && n=$((n+1)) ||:
 %config(noreplace) %{_sysconfdir}/rpc
 %config(missingok,noreplace) %{_sysconfdir}/ld.so.cache
 %config %{_sysconfdir}/locale-gen.conf
+%exclude /%{_lib64}/libcrypt.so.1 
```

Comment:
if the line below hardcodes /lib64 in the glob, this exclude should probably hardcode /lib64 as well right? Or, change the line below to /%{_lib64}, but I don't know if that's a good idea for a stable release change